src/EnergyPlus/DXCoils.cc

```diff
@@ -17312,6 +17312,7 @@ void CalcVRFHeatingCoil_FluidTCtrl(EnergyPlusData &state,
                                                    state.dataDXCoils->DXCoil(thisDXCoil.CompanionUpstreamDXCoil).CoolingCoilRuntimeFraction));
         }
 
+        OutletAirEnthalpy = min(OutletAirEnthalpy, FullLoadOutAirEnth);
```

Comment:
Why? Is this model not smart enough to calculate this correctly in the first place? There are 2 modes of operation in a coil. CyclingFan, or ConstantFan. For CyclingFan mode, the outlet condition is the full outlet condition and the air mass flow rate is the average. For ConstantFan mode, the outlet condition is the average of compressor off condition (the inlet) and compressor on condition (the outlet at full capacity). So what this says is I don't care what fan cycling mode is used, the model will just output the full load outlet enthalpy, which will always be lower than the average enthalpy based on PLR. So no, please do not do this.